vhdbuilder/packer/vhd-image-builder-mariner2-arm64.json

```diff
@@ -1,404 +1,409 @@
-          "destination": "/home/packer/NOTICE.txt"
-        },
-        {
-          "type": "shell",
-          "inline": [
-            "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} TELEPORTD_PLUGIN_DOWNLOAD_URL={{user `teleportd_plugin_download_url`}} ENABLE_FIPS={{user `enable_fips`}} SGX_INSTALL={{user `sgx_install`}} /bin/bash -ux /home/packer/pre-install-dependencies.sh",
-            "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} TELEPORTD_PLUGIN_DOWNLOAD_URL={{user `teleportd_plugin_download_url`}} ENABLE_FIPS={{user `enable_fips`}} /bin/bash -ux /home/packer/install-dependencies.sh"
-          ]
-        },
-        {
-          "type": "shell",
-          "inline": "sudo reboot",
-          "expect_disconnect": true,
-          "skip_clean": true,
-          "pause_after": "60s"
-        },
-        {
-          "type": "shell",
-          "inline": [
-            "sudo FEATURE_FLAGS={{user `feature_flags`}} BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} TELEPORTD_PLUGIN_DOWNLOAD_URL={{user `teleportd_plugin_download_url`}} ENABLE_FIPS={{user `enable_fips`}} SGX_INSTALL={{user `sgx_install`}} /bin/bash -ux /home/packer/post-install-dependencies.sh"
-          ]
-        },
-        {
-          "type": "file",
-          "source": "vhdbuilder/packer/list-images.sh",
-          "destination": "/home/packer/list-images.sh"
-        },
-        {
-          "type": "shell",
-          "inline": [
-            "sudo BUILD_NUMBER={{user `build_number`}} BUILD_ID={{user `build_id`}} COMMIT={{user `commit`}} HYPERV_GENERATION={{user `hyperv_generation`}} CONTAINER_RUNTIME={{user `container_runtime`}} ENABLE_FIPS={{user `enable_fips`}} /bin/bash -ux /home/packer/list-images.sh"
-          ]
-        },
-        {
-          "type": "file",
-          "direction": "download",
-          "source": "/opt/azure/containers/image-bom.json",
-          "destination": "image-bom.json"
-        },
-        {
-          "type": "file",
-          "direction": "download",
-          "source": "/opt/azure/vhd-install.complete",
-          "destination": "release-notes.txt"
-        },
-        {
-          "type": "shell",
-          "inline": [
-            "sudo /bin/bash -eux /home/packer/cis.sh",
-            "sudo /bin/bash -eux /home/packer/cleanup-vhd.sh",
-            "sudo /usr/bin/waagent -force -deprovision+user && export HISTSIZE=0 && sync || exit 125"
-          ]
-        }
-    ]
-  }
\ No newline at end of file
+  "variables": {
```

Comment:
sorry for huge diff but this indentation has been killing my search and replace abilities